python/ray/worker.py

```diff
@@ -1218,6 +1219,10 @@ def connect(node,
     )
     if job_config is None:
         job_config = ray.job_config.JobConfig()
+
+    if mode == SCRIPT_MODE:
+        ray_pkg.update_runtime_env(job_config)
```

Comment:
Since this code only needs to run for the driver, why not place this code in init()? Then checking the mode will not be needed.

Reply:
It's a good idea to put it there, but we still need to do this, since there is also LOCAL_MODE